.travis.yml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright © 2016 Cask Data, Inc.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+language: java
+
+jdk:
+  - oraclejdk7
+
+branches:
+  only:
+    - develop
+    - /^release\/.*$/
+
+script: mvn -Dhadoop.version="$HADOOP_VERSION" test -fae -P examples,templates,unit-tests -Dtwill.zk.server.localhost=false
```

Comment:
Does ``-Dtwill.zk.server.localhost`` need a comment? I could not find that in BUILD.rst.